src/Datasource/EntityTrait.php

```diff
@@ -696,6 +696,7 @@ public function clean()
         $this->_dirty = [];
         $this->_errors = [];
         $this->_invalid = [];
+        $this->_original = [];
```

Comment:
Can we get a test that ensures `getOriginal()` doesn't return data after clean() has been called?